### DISCUSSION TITLE
Open osmand by default

### DISCUSSION THREAD

**Discussion by QuestionsForThose** [osmandapp/OsmAnd#17198] [General]
Using pixel 4a5g when using google contacts how can I set it so that it opens osmand rather than google maps when clicking through someone's address

**Comment by tommycrock** [comment]
I don't know of a way to go from address automatically to osmand as there isn't a default map/navigation app setting AFAIK. If you wanted you could add the link, copyable from osmand in the form https://osmand.net/map?pin=lat,lon#zoom/lat/lon, as one of the contact's websites. Obviously that takes upfront legwork.
